connection.go

```diff
@@ -56,6 +56,7 @@ type snowflakeConn struct {
 	ctx             context.Context
 	cfg             *Config
 	rest            *snowflakeRestful
+	internal        InternalClient
```

Comment:
reflection on any snowflake connection for the field name `internal` will get this